<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>274</width>
    <height>568</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_4" stretch="0,0,0">
   <item>
    <widget class="QGroupBox" name="group_curve_selection">
     <property name="title">
      <string>Select Curve</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout" stretch="3">
      <item>
       <widget class="QListWidget" name="curve_list"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="group_summary">
     <property name="sizePolicy">
      <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>250</width>
       <height>200</height>
      </size>
     </property>
     <property name="title">
      <string>Fit Summary</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QTreeWidget" name="summary_tree">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="uniformRowHeights">
         <bool>false</bool>
        </property>
        <attribute name="headerDefaultSectionSize">
         <number>80</number>
        </attribute>
        <column>
         <property name="text">
          <string>Property</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Value</string>
         </property>
        </column>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="group_parameters">
     <property name="sizePolicy">
      <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>250</width>
       <height>200</height>
      </size>
     </property>
     <property name="title">
      <string>Parameter Details</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QTreeWidget" name="param_tree">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <attribute name="headerDefaultSectionSize">
         <number>80</number>
        </attribute>
        <column>
         <property name="text">
          <string>Parameter</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Value</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Std</string>
         </property>
        </column>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
